manim/__main__.py

```diff
@@ -135,7 +135,7 @@ def get_module(file_name):
             sys.exit(2)
     else:
         if os.path.exists(file_name):
-            module_name = re.sub(r"\..+$", "", file_name.replace(os.sep, "."))
+            module_name = file_name[:-3].replace(os.sep, '.').split('.')[-1]
```

Comment:
Perhaps add a line before this to check that `module_name[-3:] == ".py"`. Also, I think @PgBiel has Opinions about this line.

Reply:
Will do, but isn't it sort of mandatory for the file to be a python file anyway? Should I make it raise an exception if itsnt a file ending with `.py`?

Reply:
"Trust, but verify" ;)

Reply:
Understood :)

Also done in the latest commit.